project-generator/src/main/java/com/redhat/ipaas/project/converter/DefaultProjectGenerator.java

```diff
@@ -109,6 +115,20 @@ public DefaultProjectGenerator(ConnectorCatalog connectorCatalog) {
         );
     }
 
+
+    public byte[] generatePom(Connector connector) throws IOException {
+        Set<MavenGav> connectors = new LinkedHashSet<>();
+        String[] splitGav = connector.getCamelConnectorGAV().get().split(":");
+        if (splitGav.length == 3) {
```

Comment:
Same thing can be said for the integrations pom logic.  Best keep it uniform for now.